collector/index.md

```diff
@@ -0,0 +1,425 @@
+      <td>as above, but actual contents of the certificate(s)</td>
+    </tr>
+    <tr>
+      <td>db_sslcert (DB_SSLCERT)</td>
+      <td>[none]</td>
+      <td>file name of the client SSL certificate</td>
+    </tr>
+    <tr>
+      <td>db_sslcert_contents (DB_SSLCERT_CONTENTS)</td>
+      <td>[none]</td>
+      <td>as above, but actual contents of the certificate</td>
+    </tr>
+    <tr>
+      <td>db_sslkey (DB_SSLKEY)</td>
+      <td>[none]</td>
+      <td>location for the secret key used for the client certificate</td>
+    </tr>
+    <tr>
+      <td>db_sslkey_contents (DB_SSLKEY_CONTENTS)</td>
+      <td>[none]</td>
+      <td>as above, but actual contents of the key</td>
+    </tr>
+  </tbody>
+</table>
+
+## AWS settings
+
+Only relevant if you are running your database in AWS RDS. Note that the <code>aws_endpoint_*</code>
+settings are only relevant if you are using custom AWS endpoints. See
+[the AWS documentation](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/) for details.
+
+<table>
+  <thead>
+    <tr>
+      <th>setting</th>
+      <th>default</th>
+      <th>description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>aws_region (AWS_REGION)</td>
+      <td>n/a, required if aws</td>
+      <td>region your AWS server is running in</td>
+    </tr>
+    <tr>
+      <td>aws_db_instance_id (AWS_INSTANCE_ID)</td>
+      <td>n/a, required if aws</td>
+      <td>the instance id of your AWS server</td>
+    </tr>
+    <tr>
+      <td>aws_access_key_id (AWS_ACCESS_KEY_ID)</td>
+      <td>n/a, required if aws and using logs?</td>
+      <td>access key id of IAM role with permissions to?</td>
+    </tr>
+    <tr>
+      <td>aws_secret_access_key (AWS_SECRET_ACCESS_KEY)</td>
+      <td>n/a, required if aws and using logs?</td>
+      <td>secret access key of IAM role with permissions to?</td>
+    </tr>
+    <tr>
+      <td>aws_endpoint_signing_region (AWS_ENDPOINT_SIGNING_REGION)</td>
+      <td>[none]</td>
+      <td>?</td>
+    </tr>
+    <tr>
+      <td>aws_endpoint_rds_url (AWS_ENDPOINT_RDS_URL)</td>
+      <td>[none]</td>
+      <td>?</td>
+    </tr>
+    <tr>
+      <td>aws_endpoint_ec2_url (AWS_ENDPOINT_EC2_URL)</td>
+      <td>[none]</td>
+      <td>?</td>
+    </tr>
+    <tr>
+      <td>aws_endpoint_cloudwatch_url (AWS_ENDPOINT_CLOUDWATCH_URL)</td>
+      <td>[none]</td>
+      <td>?</td>
+    </tr>
+    <tr>
+      <td>aws_endpoint_cloudwatch_logs_url (AWS_ENDPOINT_CLOUDWATCH_LOGS_URL)</td>
+      <td>[none]</td>
+      <td>?</td>
+    </tr>
+  </tbody>
+</table>
+
+## Azure settings
+
+Only relevant if you are running your database in Azure.
+
+<table>
+  <thead>
+    <tr>
+      <th>setting</th>
+      <th>default</th>
+      <th>description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>azure_db_server_name (AZURE_DB_SERVER_NAME)</td>
+      <td>n/a, required if azure?</td>
+      <td>?</td>
+    </tr>
+    <tr>
+      <td>azure_eventhub_namespace (AZURE_EVENTHUB_NAMESPACE)</td>
```

Comment:
We could reference the Log Insights documentation here (which explains how to use this setting)